app/views/nominations/request_nomination_invite/review.html.erb

```diff
@@ -14,6 +14,10 @@
       <%= EmailDecorator::new(@nomination_request_form.school.primary_contact_email).partial_email %>
     </p>
 
+    <p class="govuk-body">
+      We've redacted the email address held in the GIAS register for data security reasons. It may belong to a staff member, or many schools use a shared email address, starting with words like 'admin', 'office', 'info' or 'headteacher'.
```

Comment:
```suggestion
      We've redacted the email address held in the GIAS register for data security reasons. It may belong to a staff member, or many schools use a shared email address starting with words like 'admin', 'office', 'info' or 'headteacher'.
```